aten/src/ATen/core/interned_strings.h

```diff
@@ -220,6 +220,7 @@ namespace c10 {
   _(onnx, Constant)                  \
   _(onnx, ConstantFill)              \
   _(onnx, Div)                       \
+  _(onnx, Einsum)                    \
```

Comment:
I suppose this is no longer needed?

Reply:
I asked myself the same question. What is the policy for adding interned strings? Should every onnx op be present here as they are added to onnx spec or just when they are being referenced by code?